src/clusterfuzz/_internal/cron/triage.py

```diff
@@ -379,7 +427,10 @@ def main():
     # Check if the crash is important, i.e. it is either a reproducible crash
     # or an unreproducible crash happening frequently.
     if not _is_crash_important(testcase):
-      continue
+      # Check if the crash is a startup crash, i.e. it is causing the fuzzer
+      # to crash on startup and not allowing the fuzzer to run longer
+      if not _is_libfuzzer_android_startup_crash(testcase):
```

Comment:
Can we rearrange this logic to check that this is on the android platform. Something like:

```
def _is_libfuzzer_android_startup_crash(testcase): # <--- Rename to something more indicative? (ex. is_blocking_progress)
    """Checks if the given testcase is an android libfuzzer crash"""
    ...

def is_crash_important_android(testcase):
    if _is_libfuzzer_android_startup_crash:
        # important

if not _is_crash_important(testcase):
    if testcase.platform == android and not is_crash_important_android(testcase):
        continue
```

Reply:
I have updated the logic as per your request. Please take a look.